clustered/server/src/main/java/org/ehcache/clustered/server/EhcacheActiveEntity.java

```diff
@@ -279,6 +294,13 @@ public void disconnected(ClientDescriptor clientDescriptor) {
 
   @Override
   public EhcacheEntityResponse invoke(ClientDescriptor clientDescriptor, EhcacheEntityMessage message) {
+    if(this.offHeapResourceIdentifiers.isEmpty()) {
+      return responseFactory.failure(
+          new IllegalStateException("Server started without any offheap resources defined." +
```

Comment:
`ResourceConfigurationException` sounds better to me. Let's stop using any form of `RuntimeException` across the wire, please. Those are a pain to handle on the client side.